docs/quick-start/transmitters/tx-aion-internal.md

```diff
@@ -62,5 +62,47 @@ Wait for the Lua script screen to close the "WiFi Running" screen and your modul
 
 Verify the version and hash in the main screen of ExpressLRS Lua script (you will first need to close it and relaunch the script).
 
+## HELP! I think I bricked my internal module!
+
+Okay, breathe. The module is recoverable.
+
+You will need a couple of files:
+
+1. An EdgeTX binary that allows you to use the USB port on the radio to reflash the module. Download it from [here](../../assets/recovery/tpro-ada778ee4.bin) (Right-click, Save-as) and save it into your Jumper T-Pro SD Card `/firmwares` folder.
+2. A zip file with all the recovery scripts and firmwares. Download it from [here](../../assets/recovery/jumper-recover.zip) (Right-click, Save-as) and extract it into a folder in your harddrive.
+
+### Procedure
+
+With the EdgeTX binary in your `/firmwares` SD Card folder, reboot the radio into DFU/Bootloader mode. Write the firmware into it and reboot to ensure it got written. One way to check is to plug-in a USB cable and a new menu item should be available to you: USB Serial (Debug). You will need this menu item in the next steps.
```

Comment:
`FIRMWARE`

Reply:
Do you want to explain how to get into DFU/Bootloader mode i.e.
While pressing both the trim switches toward the power button, press the power button.